src/assets/icons/Icon.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from "react";
+import Apple from "./Apple";
+import Facebook from "./Facebook";
+import Logo from "./Logo";
+import Menu from "./Menu";
+import Twitter from "./Twitter";
+
+type IconProps = {
+  name: string;
+};
+
+const Icon: FC<IconProps> = ({ name }) => {
+  switch (name) {
+    case "apple":
+      return <Apple />;
+
+    case "facebook":
+      return <Facebook />;
+
+    case "logo":
+      return <Logo />;
+
+    case "menu":
+      return <Menu />;
+
+    case "twitter":
+      return <Twitter />;
+
+    default:
+      return <></>;
+  }
+};
```

Comment:
I'm not sure about this `switch` because we could have a lot of different icons and maybe this is not the perfect approach. Maybe another idea could be if you have the icons in an object and use the name to do something like `icons[name]`

Reply:
Looks waaay cleaner following your suggestion. I have adjusted it accordingly. Thanks!